swift/extractor/SwiftBuiltinSymbols.h

```diff
@@ -69,5 +70,12 @@ constexpr std::array swiftBuiltins = {
     "Vec64xFPIEEE64",
     "buildDefaultActorExecutorRef",
     "buildMainActorExecutorRef",
+    "UnsafeValueBuffer",
+    "AnyObject",
+};
+
+std::vector<std::string> objcBuiltins {
```

Comment:
Hmm, apparently the DB-CHECKs are still failing for these new builtins. 

Btw, is there a reason you went for std::vector instead of a constexpr array? This is still a header, so static and/or constexpr sounds more appropriate.

Reply:
> Hmm, apparently the DB-CHECKs are still failing for these new builtins.

jfyi, I'm looking into it now

Reply:
> is there a reason you went for std::vector instead of a constexpr array?

Because the two arrays would actually have different template types (`std::array<T, N>` where T = `std::string` or `const char *` or whatever in both cases, but `N` = the length of the array, which differs).  That makes passing it to a function a pain as the function would have to be a template on `N` (unless I'm missing a trick).

Reply:
I believe you can monomorphize various types of arrays into a std::span?